vmdb/spec/requests/api/vms_spec.rb

```diff
@@ -39,72 +43,52 @@ def app
     let(:acct2_url)            { "#{vm_accounts_url}/#{acct2.id}" }
     let(:vm_accounts_url_list) { [acct1_url, acct2_url] }
 
-    context "query VM accounts subcollection with no related accounts" do
-      before do
-        api_basic_authorize
+    it "query VM accounts subcollection with no related accounts" do
+      api_basic_authorize
 
-        run_get vm_accounts_url
-      end
+      run_get vm_accounts_url
 
-      it "empty_query_result" do
-        expect_empty_query_result(:accounts)
-      end
+      expect_empty_query_result(:accounts)
     end
 
-    context "query VM accounts subcollection with two related accounts" do
-      before do
-        api_basic_authorize
-        # create resources
-        acct1
-        acct2
+    it "query VM accounts subcollection with two related accounts" do
```

Comment:
you had `context "with two related accounts"`

was expecting to see you change this to:

```ruby
context "with two related accounts" do
  before do
    # ensure accounts exist
    acct1
    acct2
  end
  it "queries VM account subcollection" do
    api_basic_authorize
    run_get vm_accounts_url
  end
end
```